app/models/collection.rb

```diff
@@ -2,6 +2,14 @@
 
 # Generated by hyrax:models
 class Collection < ActiveFedora::Base
+  property :bulkrax_identifier, predicate: ::RDF::URI("https://hykucommons.org/terms/bulkrax_identifier"), multiple: false do |index|
```

Comment:
Does this need to be added to CollectionResource as well?

Reply:
I think all of the resources should likely include this `bulkrax_identifier`; I'll work on adding that.